ios/brave-ios/Sources/BraveWallet/Extensions/BitcoinWalletServiceExtensions.swift

```diff
@@ -36,4 +36,25 @@ extension BraveWalletBitcoinWalletService {
       }
     )
   }
+
+  /// - Parameters:
+  ///     - accountId: A list of `BraveWallet.AccountId`
+  ///
+  /// - Returns: The BTC balance of the given `BraveWallet.AccountId` in `Double`; Will return a nil if there is an issue fetching balance.
+  func fetchBTCBalance(accountId: BraveWallet.AccountId) async -> Double? {
+    guard let btcBalance = await self.balance(accountId: accountId).0
+    else { return nil }
+
+    let availableSatoshiString = String(btcBalance.availableBalance)
+    let formatter = WeiFormatter(decimalFormatStyle: .decimals(precision: 8))
+    if let valueString = formatter.decimalString(
+      for: availableSatoshiString,
+      radix: .decimal,
+      decimals: 8
+    ) {
+      return Double(valueString)
+    } else {
+      return nil
+    }
+  }
```

Comment:
In most contexts we need `total balance`, in a context of send screen we need `available balance`. For desktop ui there is also a popup which shows all three lines of complex BTC balance. 
FYI

Reply:
i will update this using `totalBalance`. we have a separate issue to integrate bitcoin balance full info https://github.com/brave/brave-browser/issues/37640
will keep in mind to use `availableBalance` in send tx. 
thanks!